kotsadm/web/src/components/gitops/GitOpsDeploymentManager.jsx

```diff
@@ -113,21 +101,57 @@ class GitOpsDeploymentManager extends React.Component {
     }
   }
 
+  getGitopsState = async () => {
+    const freshGitops = await this.getGitops()
+
+    if (freshGitops.enabled) {
```

Comment:
since it can `undefined`
```suggestion
    if (freshGitops?.enabled) {
```